### DISCUSSION TITLE
Access blocked: GYB has not completed the Google verification process

### DISCUSSION THREAD

**Discussion by TheBigForrest** [GAM-team/got-your-back#479] [General]
I am getting the following error:

Access blocked: GYB has not completed the Google verification process

GYB has not completed the Google verification process. The app is currently being tested, and can only be accessed by developer-approved testers. If you think you should have access, contact the [developer](https://accounts.google.com/).
If you are a developer of GYB, see [error details](https://accounts.google.com/).
Error 403: access_denied

As an additional note, the instructions for using GYB are a bit complicated.  I humbly suggest creating a YouTube video to show how to make it work.

**Comment by edwinfinch** [comment]
I also ran into this issue, following these steps and trying again resolved it for me:

https://stackoverflow.com/a/75466700

**Reply by isaacgonzalez77** [reply]
I had the same error ...doing those steps sort of helped as I published my the app under "audience". Since it's Free Gmail there was no "users" to add though since internal user you can't use unless it's Gworkspace, so it's external users only. I then just use read only access from command line option for backing up instead of full permission and this did the trick...whew I was worried that GYB doesn't work anymore for free Gmail...Instructions could use some refreshing due to this, because in the past it worked without a hitch...
